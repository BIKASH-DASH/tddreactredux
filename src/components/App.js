import React, { Component } from 'react'

export default class App extends Component {
    render() {
        return (
            <div>
                <h2>loot check</h2>
            </div>
        )
    }
}
