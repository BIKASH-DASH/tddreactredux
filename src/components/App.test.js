import React from 'react'
import Enzyme,{shallow} from 'enzyme'
import App from './App'
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
const wrapper = shallow(<App/>);

describe('Name of the group', () => {
    
    it('should render properly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});