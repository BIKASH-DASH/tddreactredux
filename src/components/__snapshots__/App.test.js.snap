// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name of the group should render properly 1`] = `ShallowWrapper {}`;
